xbmc/cores/dvdplayer/DVDCodecs/DVDFactoryCodec.cpp

```diff
@@ -216,6 +216,8 @@ CDVDVideoCodec* CDVDFactoryCodec::CreateVideoCodec(CDVDStreamInfo &hint, const C
       case AV_CODEC_ID_MPEG4:
       case AV_CODEC_ID_MSMPEG4V2:
       case AV_CODEC_ID_MSMPEG4V3:
+      case AV_CODEC_ID_MPEG1VIDEO:
+      case AV_CODEC_ID_MPEG2VIDEO:
```

Comment:
this is btw some part of the code where I agree to @jimfcarroll 's di. platform dependent code could register a decoder here and factory is free of ifdefs.